android/anc/src/test/java/org/smartregister/fhircore/anc/util/CalculationUtilsTest.kt

```diff
@@ -22,9 +22,9 @@ import org.junit.Test
 class CalculationUtilsTest {
 
   @Test
-  fun testBMI_viaStandardUnit() {
+  fun testBMI_viaUSCUnit() {
```

Comment:
Rename this test method as per the convention e.g `testBMIViaUSCUnitIsComputed`

Reply:
done